framework/include/utils/MooseUtils.h

```diff
@@ -543,6 +544,36 @@ relativeFuzzyLessThan(const T & var1,
       tol * (std::abs(MetaPhysicL::raw_value(var1)) + std::abs(MetaPhysicL::raw_value(var2)))));
 }
 
+/**
+ * @param value The quantity to test for zero-ness
+ * @param tolerance The tolerance for testing zero-ness. The default is 1e-18
```

Comment:
isnt it -24? or is TOLERANCE not the same as libmesh::TOLERANCE?
I see REL_TOLERANCE being defined to 1e-6 not TOLERANCE

Reply:
I should clarify the comment that it is 1e-18 for double precision
```c++
#ifdef LIBMESH_DEFAULT_SINGLE_PRECISION
static constexpr Real TOLERANCE = 2.5e-3;
# if defined (LIBMESH_DEFAULT_TRIPLE_PRECISION) || \
     defined (LIBMESH_DEFAULT_QUADRUPLE_PRECISION)
#  error Cannot define multiple precision levels
# endif
#endif

#ifdef LIBMESH_DEFAULT_TRIPLE_PRECISION
static constexpr Real TOLERANCE = 1.e-8;
# if defined (LIBMESH_DEFAULT_QUADRUPLE_PRECISION)
#  error Cannot define multiple precision levels
# endif
#endif

#ifdef LIBMESH_DEFAULT_QUADRUPLE_PRECISION
static constexpr Real TOLERANCE = 1.e-11;
#endif

#if !defined (LIBMESH_DEFAULT_SINGLE_PRECISION) && \
    !defined (LIBMESH_DEFAULT_TRIPLE_PRECISION) && \
    !defined (LIBMESH_DEFAULT_QUADRUPLE_PRECISION)
static constexpr Real TOLERANCE = 1.e-6;
#endif
```

Reply:
sounds good.
I must have pulled the wrong one when I checked